webapp/src/main/java/life/qbic/views/register/UserRegistrationHandler.java

```diff
@@ -94,7 +95,12 @@ public void onSuccess() {
   }
 
   @Override
-  public void onFailure(String reason) {
+  public void onUserRegistrationFailed(UserRegistrationException exception) {
+
+  }
+
+  @Override
+  public void onUserRegistrationFailed(String reason) {
```

Comment:
do we need both?

Reply:
Ah no, we need it for Unexpected Failures. But I renamed it now to make its intention clear